airbyte-integrations/connectors/source-marketo/source_marketo/schemas/leads.json

```diff
@@ -1,280 +1,571 @@
 {
-  "type": ["object", "null"],
+  "type": [
+    "object",
+    "null"
+  ],
   "additionalProperties": false,
   "properties": {
     "company": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "site": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "billingStreet": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "billingCity": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "billingState": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "billingCountry": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "billingPostalCode": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "website": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "mainPhone": {
-      "type": ["string", "null"]
+      "type": [
+        "string",
+        "null"
+      ]
     },
     "annualRevenue": {
-      "type": ["number", "null"]
+      "type": [
+        "string",
+        "number",
```

Comment:
1. I just noticed this schema change. Please note that this is a backwards incompatible type change. Anyone currently using marketo will begin to have their syncs break because we are sending strings in columns which were declared as numbers before. 
2. I'm really unsure about why we are changing this to a string. `annualRevenue` doesn't sound like something that would be a string, definitely not a `false` like the user is reporting. So we are incurring a very expensive breaking change to all marketo users without having proof of the hypothesis. I think we should not change this schema of leads and instead release a `-beta` version of the connector which contains a lot of debugging information, basically log everything, and ask the user to try it to help us understand the issue

Also a note on the usage of `type` array here: it means the type will either be string, number, or null. Generally speaking this is not something that destinations can handle e.g: most data warehouses don't have a concept of a column being "either" string or number. We should generally only have two things in that array `null` and the actual data type. Maybe we should create a SAT validation for this.